alvr/client_core/src/graphics/staging.rs

```diff
@@ -118,6 +118,8 @@ impl StagingRenderer {
                 ck!(gl.use_program(Some(self.program)));
 
                 ck!(gl.viewport(0, 0, self.viewport_size.x, self.viewport_size.y));
+                ck!(gl.disable(gl::SCISSOR_TEST));
+                ck!(gl.disable(gl::STENCIL_TEST));
```

Comment:
Huh, could've bet that was the issue but I failed to debug it so I just accepted your split renderer pr